otbox.py

```diff
@@ -97,6 +99,43 @@ def __init__(self):
         )
         self.mqttthread.start()
 
+
+    def set_params(self):
+        # sets the value of BAUDRATE, BROKER_ADDRESS, MOTE_USB_DEVICES, FIRMWARE_EUI64_RETRIEVAL, FIRMWARE_TEMP
+        # based on the selected testbed
+        global BAUDRATE
```

Comment:
Any specific reason these vars are global?

Reply:
No particular reason. They are global variables in the current version of `otbox.py` so I left it as it is. I'll change them to class member variables, as requested

Reply:
Sounds good!

Reply:
Actually, on further inspection, I see that BAUDRATE variable is used within `SerialportHandler` class as well. Thus, it's best to keep it global. The rest can go as class members